src/components/uielements/poolStep/PoolStatus.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { Row } from 'antd'
+import BigNumber from 'bignumber.js'
+import { useIntl } from 'react-intl'
+
+import Label from '../label'
+import Trend from '../trend'
+import { PoolStatusWrapper } from './PoolStatus.style'
+
+type Props = {
+  amount: BigNumber
+  asset: string
+  label: string
+  target: string
+  trend: BigNumber
+}
+
+const PoolStatus: React.FC<Props> = (props: Props): JSX.Element => {
+  const { amount, asset, label, target, trend } = props
+  const intl = useIntl()
+  const amountVal = intl.formatNumber(amount.toNumber(), { currency: 'USD', style: 'currency' })
```

Comment:
If we are using `AssetAmount` we can use a format helper [`formatAssetAmount`](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/asset.ts#L70) or [`formatAssetAmountCurrency`](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/asset.ts#L75-79). It depends on what you want to show here.